build-system/npm-publish/utils.js

```diff
@@ -26,6 +26,17 @@ function getComponents() {
   return components;
 }
 
+/**
+ *
+ * @return {{name: string, version: string}}
+ */
+function getCoreConfig() {
```

Comment:
tried to keep with the pattern of configs for each of our packages, seemed marginally better than hardcoding it in the yml file directly

Reply:
sg. can this just be a const object instead of a function?

Reply:
changed